packages/firestore/src/util/log.ts

```diff
@@ -15,11 +15,11 @@
  * limitations under the License.
  */
 
-import { Logger, LogLevel } from '@firebase/logger';
+import { Logger, LogLevel, LogLevelString } from '@firebase/logger';
 import { SDK_VERSION } from '../core/version';
 import { PlatformSupport } from '../platform/platform';
 
-export { LogLevel };
+export { LogLevel, LogLevelString };
```

Comment:
You probably don't need to export these anymore. Instead, you can rely on the fact that our typings now match the types in `@firebase/logger`.